_layouts/base_slides.html

```diff
@@ -86,10 +86,12 @@
 </div>
 {% endif %}
 
-<div class="footnote" style="bottom: 3em;">{% icon last_modification %} {{locale['last-modification'] | default: "Updated"}}: {{ page.last_modified_at | date: "%b %-d, %Y"}}</div>
+<div class="footnote" style="bottom: 4em;">{% icon last_modification %} {{locale['last-modification'] | default: "Updated"}}: {{ page.last_modified_at | date: "%b %-d, %Y"}}</div>
 
 
-<div class="footnote" style="bottom: 2em;">{% icon text-document %}<a href="slides-plain{% if page.lang %}_{{page.lang | upcase }}{% endif %}.html"> {{locale['plaintext-slides'] | default: "Or view the plain-text slides without JS"}}</a></div>
+<div class="footnote" style="bottom: 3em;">{% icon text-document %}<a href="slides-plain{% if page.lang %}_{{page.lang | upcase }}{% endif %}.html"> {{locale['plaintext-slides'] | default: "Or view the plain-text slides without JS"}}</a></div>
+
+<div class="footnote" style="bottom: 2em;"><strong>{{locale['first-time-using-slides'] | default: "First time using slides? Use the arrows on your keyboard to move between them!"}}</div>
```

Comment:
We could just remove the "first time using slides?" part I think. It will be more concise, and this is just a good reminder even for those who have used the slide decks before.What do you think?

We could also consider using icon to make it more visible, something like:

![image](https://user-images.githubusercontent.com/2563865/140291792-66b595fc-f19b-46bb-adfc-bf5f212757e4.png)